src/commands/createFunctionApp/stacks/FunctionAppStackStep.ts

```diff
@@ -35,6 +37,7 @@ export class FunctionAppStackStep extends AzureWizardPromptStep<IFunctionAppWiza
                 break;
             }
         }
+
```

Comment:
Was this intended?

Reply:
Yeah, wanted to give it some space to look neater.